index.html

```diff
@@ -14,6 +14,9 @@
     </div>
     <div id="subtitle">
     </div>
+    <div class="loader"></div>
+    <div class="message_portion"></div>
```

Comment:
please make this message-portion. snakecase in css is not the right convention.

Reply:
> please make this message-portion. snakecase in css is not the right convention.

ok..working on it